privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -0,0 +1,208 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Brands
+# Dev Note: When adding to this section, use this file for naming conventions: https://github.com/mozilla/bedrock/blob/master/l10n/en/brands.ftl
+
+-brand-name-firefox = Firefox
+-brand-name-firefox-relay = Firefox Relay
+-brand-name-relay = Relay
+-brand-name-firefox-account = Firefox Account
+-brand-name-firefox-browser = Firefox Browser
+-brand-name-mozilla = Mozilla
+-brand-name-firefox-lockwise = Firefox Lockwise
+-brand-name-firefox-monitor = Firefox Monitor
+-brand-name-pocket = Pocket
+
+## Meta Data
+
+meta-title = { -brand-name-firefox-relay }
+meta-description = { -brand-name-firefox-relay } makes it easy to create email aliases, aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.
+
+## Header 
+
+logo-alt= { -brand-name-firefox-relay }
+nav-menu = Menu
+nav-home = Home
+
+# FAQ stands for Frequently Asked Questions. The intent of this page is to answer commonly asked questions.
+nav-faq = FAQ
+nav-profile-sign-in = Sign In
+nav-profile-manage-fxa = Manage your { -brand-name-firefox-account }
+nav-profile-sign-out = Sign Out
+nav-profile-sign-out-relay = Sign Out of { -brand-name-relay }
+nav-profile-sign-out-confirm = Are you sure you want to sign out?
+
+## Footer
+
+nav-footer-privacy = Privacy
+nav-footer-relay-terms = { -brand-name-relay } Terms
+nav-footer-legal = Legal
+
+## Bento Menu
+
+bento-button-title = { -brand-name-firefox } apps and services
+fx-makes-tech = { -brand-name-firefox } is tech that fights for your online privacy.
+made-by-mozilla = Made by { -brand-name-mozilla }
+fx-desktop = { -brand-name-firefox-browser } for Desktop
+fx-lockwise = { -brand-name-firefox-lockwise }
+fx-mobile = { -brand-name-firefox-browser } for Mobile
+fx-monitor = { -brand-name-firefox-monitor }
+fx-pocket = { -brand-name-pocket }
+mobileCloseBentoButtonTitle = Close menu
+
+## Home Page - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/
```

Comment:
This is not needed. The product staging info will provided in Pontoon on project dashboard.

Reply:
Cool. I pulled this convention from the Bedrock project. Should I remove all of the URLs?

Reply:
Removed!